pkgs/applications/networking/browsers/chromium/common.nix

```diff
@@ -513,7 +508,7 @@ let
           revert = true;
         })
       ]
-      ++ lib.optionals (chromiumVersionAtLeast "131" && stdenv.hostPlatform.isAarch64) [
+      ++ lib.optionals stdenv.hostPlatform.isAarch64 [
         # Reverts decommit pooled pages which causes random crashes of tabs on systems
         # with page sizes different than 4k. It 'supports' runtime page sizes, but has
```

Comment:
When can this patch be removed? Will upstream ever handle it?
I was not there when it was added so I don't really know a lot about this.